projects/components/src/tabs/navigable/navigable-tab-group.component.ts

```diff
@@ -14,17 +14,15 @@ import { NavigableTabComponent } from './navigable-tab.component';
       <nav mat-tab-nav-bar *htLetAsync="this.activeTab$ as activeTab" disableRipple>
         <ng-container *ngFor="let tab of this.tabs">
           <ng-container *ngIf="!tab.hidden">
-            <ht-link [paramsOrUrl]="buildNavigationParam | htMemoize: tab">
-              <div class="tab-button" *htIfFeature="tab.featureFlags | htFeature as featureState">
-                <a mat-tab-link (click)="this.onTabClick(tab)" class="tab-link" [active]="activeTab === tab">
```

Comment:
Are you sure it is still able to set active attribute for mat-tab link? This is why I had kept the ht-link outside the mat-tab-link.

Reply:
Yep. its setting active tab correctly.

Reply:
Nice. Thank you!